.gitignore

```diff
@@ -0,0 +1,7 @@
+# ignore the savefile
+userdata.txt
+
+# ignore ALL files in ANY directory named as below
+node_modules/ 
+dist/server/
+.vscode/
```

Comment:
That is good. Now, you have to ensure, that those files really aren't going to publish on GitHub. Also, please prevent the "cache" (`.angular/cache`) to be published.